src/components/collapsibleContainer/CollapsibleContainerConnected.spec.tsx

```diff
@@ -0,0 +1,142 @@
+import * as React from 'react';
+import { Store, Provider } from 'react-redux';
+import { TestUtils } from '../../utils/TestUtils';
+import { clearState } from '../../utils/ReduxUtils';
+import { mount } from 'enzyme';
+import { IReactVaporState } from '../../ReactVapor';
+import { CollapsibleContainer } from './CollapsibleContainer';
+import { setExpandedCollapsibleContainer } from './CollapsibleContainerActions';
+import { CollapsibleContainerConnected } from './CollapsibleContainerConnected';
+import { ICollapsibleContainerOwnProps, ICollapsibleContainerProps } from './CollapsibleContainer';
+import { collapsibleContainerPossibleProps } from './CollapsibleContainerTestCommon.spec';
+import { findWhere } from 'underscore';
+import { Svg } from '../svg/Svg';
+import { LinkSvg } from '../svg/LinkSvg';
+import { Tooltip } from '../tooltip/Tooltip';
+
+
+describe('<CollapsibleContainerConnected />', () => {
+  let store: Store<IReactVaporState>;
+  let basicCollapsibleContainerProps: ICollapsibleContainerOwnProps;
+
+  beforeEach(() => {
+    store = TestUtils.buildStore();
+    basicCollapsibleContainerProps = { ...collapsibleContainerPossibleProps[0] };
+  });
+
+  afterEach(() => {
+    store.dispatch(clearState());
+  });
+
+  const mountComponentWithProps = (props: ICollapsibleContainerOwnProps) => {
+    return mount(
+      <Provider store={store}>
+        <CollapsibleContainerConnected {...props}>
+          dummy children
+        </CollapsibleContainerConnected>
+      </Provider>,
+      { attachTo: document.getElementById('App') },
+    );
+  };
+
+  it('should get what to do onMount', () => {
```

Comment:
Le nom du test devrait être qqchose comme `should not throw onMount`. Même chose pour les 2 tests d'après.

Reply:
fixed